code/core/arm9/source/Application/Settings/Json/JsonAppSettingsSerializer.cpp

```diff
@@ -20,8 +20,11 @@
 #define KEY_DISPLAY_SETTINGS_BORDER_IMAGE           "borderImage"
 
 #define KEY_RUN_SETTINGS                                    "runSettings"
+#define KEY_RUN_SETTINGS_ENABLE_JIT                         "enableJit"
 #define KEY_RUN_SETTINGS_JIT_PATCH_ADDRESSES                "jitPatchAddresses"
+#define KEY_RUN_SETTINGS_ENABLE_MAIN_RAM_ICACHE             "enableMainRamICache"
```

Comment:
```suggestion
#define KEY_RUN_SETTINGS_ENABLE_ROM_ICACHE                  "enableRomICache"
```
Also update the newly added config files accordingly